tests/tests_twobody/test_orbit.py

```diff
@@ -615,27 +604,6 @@ def test_from_horizons_raise_valueerror():
     )
 
 
-@pytest.mark.remote_data
-def test_orbit_from_horizons_has_expected_elements():
```

Comment:
Is this already covered by an equivalent `Ephem.from_horizons` test?

Reply:
@astrojuanlu removing this test was only decreasing coverage for `Orbit.pqw()` function which is deprecated and was throwing a deprecated function warning, I haven't checked exactly what this test is doing with `Ephem.from_horizons` but can check and confirm whether this is essential and get back to you on this.

Reply:
We should consider replacing it by something similar that checks whether `Ephem.from_horizons` for a particular body has the expected data. Kind of like an integration test.

But we might already have those. No need to repeat them.